venv/task1.py

```diff
@@ -0,0 +1,38 @@
+import connections as con
+
+rb = con.basics.join(con.ratings,
+                    con.basics.tconst == con.ratings.tconst, "inner") \
+        .withColumn('numVotes', con.ratings.numVotes.cast('integer'))
```

Comment:
what is the purpose of casting this field to integer?

Reply:
I did this to sort accurately by the numVotes.